packages/cactus-plugin-satp-hermes/src/main/typescript/.env.example

```diff
@@ -1,38 +0,0 @@
-# CREDENTIALs
```

Comment:
@RafaelAPB @brunoffmateus I'm too late with this info, but if you refactor the gateway to be a plugin that can be loaded through the API server then you get this functionality (of the API server) for free as well. 

Longer term I'd aim to refactor the gateway so that it doesn't need it's own container image (less maintenance burden) nor it's own CLI/env/file configuration parsing logic and of course a million other things like JWT basd authorization, SocketIO, gRPC, ConnectRPC, etc.

It's too late to redo the whole thing right now, but I suggest putting it on the road map for later. It's also a refactor that can be done in smaller pieces by way of implementing the necessary interfaces one by one in smaller PRs. Once all the necessary interfaces are implemented then the parts of the code that are redundant with the API server's functionality can be just deleted and the docs updated to show it using the API server.

I'm also happy to try and help with this in the future.